app/jinja_filters.py

```diff
@@ -142,6 +152,39 @@ def format_conditional_date(context, date1=None, date2=None):
     return format_date(context, date)
 
 
+@blueprint.app_template_filter()
+def calculate_offset_date(input_datetime, offset_days, day_of_week='SU'):
```

Comment:
I've changed my mind. As initially discussed I think this should accept a dictionary rather than only support days.
So along the lines of:
```
calculate_offset_date(input_datetime, offset, day_of_week='SU'):

   offset_days = offset.get('days', 0)
   offset_weeks = offset.get('weeks', 0)
   offset_years = offset.get('years', 0)
```

The filter would then look like this:
`calculate_offset_date(collection_metadata['started_at'], {'weeks'-4, 'days':1})`

Reply:
Fixed now.